UKHO.ADM/Objects/Objects.ps1

```diff
@@ -104,12 +104,21 @@ class ADDomain {
     [string]$DomainController
     [string]$DistinguishedName
     [ADOrganisationalUnit[]]$OrganisationalUnits
-    [pscredential]$Credential
+    [psCredential]$Credential
```

Comment:
Just double checking, will this change in capitalisation break anything?

Reply:
not sure where that came in, maybe the autoformating?

Reply:
To answer the question, it won't

Reply:
Ok then :)